lib/modules/dosomething/dosomething_gladiator/dosomething_gladiator.module

```diff
@@ -56,19 +46,32 @@ function _dosomething_gladiator_build_http_client() {
 function dosomething_gladiator_send_user_to_gladiator($user, $values) {
   $client = _dosomething_gladiator_build_http_client();
 
-  if (!$client) { return; }
-
-  try {
-    $response = $client->request('POST', 'users', [
-      'body' => json_encode([
+  $options = [
+  'method' => 'POST',
+  'headers' => $client['headers'],
+  'data' =>
+      json_encode([
         'id' => $user->mail,
         'term' => 'email',
         'campaign_id' => $values['nid'],
         'campaign_run_id' => $values['run_nid'],
-        ]),
-      ]);
-  } catch (GuzzleHttp\Exception\RequestException $e) {
-    watchdog('dosomething_gladiator', $e, array(), WATCHDOG_ERROR);
+      ]),
+  ];
+
+  if (!$client) { return; }
```

Comment:
ah yes, probably. but the way this is getting made now, w/o guzzle, it's probably not needed at all